isis/src/control/apps/findfeatures/findfeatures.cpp

```diff
@@ -327,15 +433,22 @@ namespace Isis{
       cnet.SetDescription(best->matcher()->name());
       cnet.SetCreatedDate(findfeatures_runtime);
       QString target = ( ui.WasEntered("TARGET") ) ? ui.GetString("TARGET") :
-                                                     best->target();
+                                                      best->target();
       cnet.SetTarget( target );
       ID pointId( ui.GetString("POINTID"), ui.GetInteger("POINTINDEX") );
-      matcher.setDebugOff();
+      // matcher.setDebugOff();  TURN ON DEBUG OUTPUT FOR NETWORK GENERATION!
```

Comment:
Why is this here? If it needs to be on just delete this line

Reply:
This indicates a behavioral change. It was explicitly turned off in the commented code and I left it in the initial PR to indicate that change. I have since removed it.